style.css

```diff
@@ -0,0 +1,363 @@
+}
+.leftmargin {
+  margin-left: 70px;
+}
+.lafunHeadT {
+  margin: 0px;
+  padding: 10px 0px 10px 10px;
+  color: white;
+}
+.sixteen {
+  width: 16px;
+  height: 16px;
+}
+.width100 {
+  width: 100%;
+}
+.flexContainer {
+  padding: 10px;
+}
+.lafund2 {
+  border-bottom: 2px solid rgba(255, 68, 34, 0.25);
+}
+.flexContainer span {
+  margin-left: 10px;
+}
+.maText {
+  color: rgb(85, 85, 85);
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+  font-size: 12px;
+  margin: 12px 0;
+}
+.lafund3 span {
+  margin: 12px 3px;
+}
+.leftmargin h5 {
+  color: black;
+  font-size: 18px;
+  font-weight: bold;
+  margin: 5px 0px;
+}
+.leftmargin span {
+  color: black;
+  font-size: 11px;
+}
+.cat1 {
+  background-image: url("Images/cat1.png");
+}
+.catCont {
+  background-size: cover;
+  min-height: 405px;
+  background-position: center;
+  display: flex;
+}
+.catFreeArea {
+  height: 80%;
+}
+.noDecoration {
+  list-style-type: none;
+  margin: 5px;
+  padding: 0px;
+}
+.artTitle {
+  font-size: 20px;
+  font-weight: bold;
+}
+.artCategory {
+  font-family: Shentox, sans-serif;
+  font-size: 13px;
+  font-weight: 500;
+}
+.catListArt {
+  border-bottom: 1px solid #eee;
+  font-size: 14px;
+  padding: 5px;
+}
+.orangeBullets {
+  list-style: none;
+  margin-left: 0px;
+  padding-inline-start: 4px;
+}
+.orangeBullets li::before {
+  content: "\2022";
+  color: rgb(255, 68, 34);
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: 0px;
+}
+.category {
+  margin: 10px;
+}
+.socialMedia {
+  background-color: #131313;
+  padding: 2.5em 0;
+}
+.socialLogos {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 0px auto;
+  max-width: 427px;
+  padding: 0px 20px;
+  width: 60%;
+}
+.socialLogo {
+  width: 40px;
+  height: 40px;
+  background-color: rgb(49, 48, 48);
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+  display: inline-flex;
+}
+.socialFo {
+  color: white;
+  display: block;
+}
+.whiteText {
+  color: #aaa;
+}
+.AboutSec {
+  display: flex;
+  flex-wrap: nowrap;
+  width: 60%;
+  justify-content: space-between;
+  margin: 10px auto;
+  padding: 10px;
+  max-width: 450px;
+}
+.About {
+  font-size: 13px;
+}
+.copyright {
+  background-color: black;
+  width: 100%;
+  padding: 10px;
+}
+.legalI {
+  margin: auto;
+  width: 60%;
+}
+@media only screen and (min-width: 700px) {
+  .justMobile {
+    display: none;
+  }
+  header {
+    display: inline-block;
+  }
+  header a {
+    display: inline-block;
+  }
+  .headers {
+    display: grid;
+    grid-template-columns: 60% 40%;
+    grid-template-areas:
+      "a b"
+      "a c";
+  }
+  .header1Cont {
+    grid-row: span 2;
+  }
+  .latestFlex {
+    justify-content: space-evenly;
+  }
+  .disp3 {
+    width: 30%;
+    flex-wrap: wrap;
+  }
+  .latfundCont {
+    justify-content: space-around;
+  }
+  .w-h100Larg {
+    width: 100% !important;
+    height: 100px !important;
+  }
+  .hideTablet {
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_